tc/core/cuda/cuda_tc_executor.h

```diff
@@ -39,7 +39,8 @@ class CudaTcExecutor : public TcExecutor<CudaBackend> {
   /// doesn't then segfault will likely occur.
   void uncheckedRun(
       const std::vector<const void*>& inputs,
-      const std::vector<void*>& outputs) const;
+      const std::vector<void*>& outputs,
+      typename CudaBackend::RuntimeInformation info = {}) const;
```

Comment:
I would explicitly initialize to `{0}`, default initialization will prob not work in the future when we add more runtime info